MerlinAU.sh

```diff
@@ -1790,7 +1795,7 @@ _RunFirmwareUpdateNow_()
 
 	# Detect ROG and pure firmware files
 	rog_file="$(ls | grep -i '_rog_')"
-	pure_file="$(ls | grep -iE '_pureubi.w|_ubi.w' | grep -iv 'rog')"
+	pure_file="$(ls | grep -iE '(_pureubi\.w|_ubi\.w|.*_nand_squashfs\.pkgtb)$' | grep -iv 'rog')"
```

Comment:
This was required to fix the matching of non .w files. For example, firmware files using: ".pkgtb"

Reply:
Didn't like we how were matching file names as well as extensions. Removed this for a newer addition found below.